### PR TITLE
Update Example02_Pipeline.cs

### PR DESCRIPTION
**Pull request by garethdebruyn**
text["LStrip"], text["RStrip"] no longer seem to be part of TextSkill

### Contribution Checklist

- [ ] The code builds clean without any errors or warnings
- [ ] The PR follows SK Contribution Guidelines (https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md)
- [ ] The code follows the .NET coding conventions (https://learn.microsoft.com/dotnet/csharp/fundamentals/coding-style/coding-conventions) verified with `dotnet format`
- [ ] All unit tests pass, and I have added new tests where possible
- [ ] I didn't break anyone :smile:

### CODE REVIEW

**Comment by shawncal** [comment]
Good call -- this is a point of confusion. `LStrip` and `RStrip` actually are part of TextSkill, just not "the" TextSkill...
They exist in this TextSkill in the KernelSyntaxExamples project:
https://github.com/microsoft/semantic-kernel/blob/main/samples/dotnet/kernel-syntax-examples/Skills/TextSkill.cs

...but not in the CoreSkills.TextSkill version:
https://github.com/microsoft/semantic-kernel/blob/main/dotnet/src/SemanticKernel/CoreSkills/TextSkill.cs

I think this PR is on the right track -- we should probably just delete the KernelSyntaxExamples.TextSkill class and have examples use the CoreSkills version. Assigning @RogerBarreto to help this PR, cleanup of KernelSyntaxExample.

File: samples/dotnet/kernel-syntax-examples/Example02_Pipeline.cs [hunk 1]
```diff
@@ -23,8 +23,8 @@ public static async Task RunAsync()
         var text = kernel.ImportSkill(new TextSkill());
 
         SKContext result = await kernel.RunAsync("    i n f i n i t e     s p a c e     ",
-            text["LStrip"],
```
Comment (adrianwyatt) [inline samples/dotnet/kernel-syntax-examples/Example02_Pipeline.cs:26]: These values come from `\samples\dotnet\kernel-syntax-examples\Skills\TextSkill.cs` which contain the `LStrip` and `RStrip` semantic functions. `TrimStart` and `TrimEnd` come from the TextSkill in the kernel's core skills. The build breaks with this current change.

**Review by adrianwyatt — CHANGES_REQUESTED** [review]

**Comment by RogerBarreto** [comment]
Added MyTextSkills class to the first examples to clarity.

We have samples (12) that uses together TextSkills from both `Examples` and `Core`, I'm currently working on something in parallel to make the Examples project easier and more welcoming.

**Review by RogerBarreto — APPROVED** [review]
Made some changes on top of his (Added MyTextSkill in the same example file for clarity)

**Review by adrianwyatt — APPROVED** [review]

**Pull request merged**